WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeSearchFragment.java

```diff
@@ -171,4 +145,17 @@ private void configureSearchView() {
         mSearchView.setQuery(mLastSearch, true);
         mSearchView.setMaxWidth(SEARCH_VIEW_MAX_WIDTH);
     }
+
+    private List<ThemeModel> filter(@NonNull List<ThemeModel> themes) {
+        if (TextUtils.isEmpty(mLastSearch)) {
+            return themes;
+        }
+        List<ThemeModel> filteredThemes = new ArrayList<>();
+        for (ThemeModel theme : themes) {
+            if (theme.getName().toLowerCase().contains(mLastSearch.toLowerCase())) {
```

Comment:
It would be more efficient to create a string variable for `mLastSearch.toLowerCase()` than to do the lower case conversion with every iteration. More importantly, I don't think matching the theme name is enough - we should match the description as well. So:

```
String lcSearch = mLastSearch.toLowerCase();
for (ThemeModel theme : themes) {
    if (theme.getName().toLowerCase().contains(lcSearch)
            || (theme.getDescription() != null && theme.getDescription().toLowerCase().contains(lcSearch))) {
        filteredThemes.add(theme);
    }
}
```